docs/source/guide/mmm/mmm_intro.md

```diff
@@ -0,0 +1,42 @@
+# Introduction to Media Mix Modeling
+
+A problem faced by many companies is how to allocate marketing budgets across different media channels. For example, how should funds be allocated across TV, radio, social media, direct mail, or daily deals?
+
+One approach might be to use heuristics, i.e. sensible rules of thumb, about what might be most appropriate for your company. For instance, a widely used approach is to simply set your marketing budget as a percentage of expected revenues. But this involves guesswork - something we want to avoid regardless of the size of the marketing budget involved.
+
+Fortunately, with Bayesian modeling, we can do better than this! So-called Media Mix Modeling (MMM) can estimate how effective each advertising channel is in driving our outcome measure of interest, whether that is sales, new customer acquisitions, or any other KPI. Once we have estimated each channel's effectiveness we can optimize our budget allocation to maximize our KPI.
```

Comment:
Would it be better to write out KPI in full here?

Reply:
Good point. Fixing in an upcoming commit